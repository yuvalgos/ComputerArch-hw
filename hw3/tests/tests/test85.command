./dflow_calc tests/test85.dat tests/test85.in d192 d92 p89 d23 d173 d110 d40 d5 p161 d50 d113 d82 p57 d87 p110 d124 d9 p80 d97 d103 p124 d78 p44 d37 d86 d37 d63 p16 d111 d121 d172 d77 d156 p12 p31 p36 d90 d130 p6 d168 p145 d34 p94 p32 d180 d80 p153 p136 p119 d60 p159 p21 d126 p51 d101 p160 d63 d143 p123 p81 p35 d183 d191 p90 p30 p165 d199 p180 d95 p8 p61 p58 d96 p189 p157 p11 d15 p92 p185 d121 d97 p105 d83 d7 p41 d40 p40 d31 d55 p155 p76 d72 p66 p44 d112 d31 p145 d135 p28 p150 d175 p41 d6 p49 d56 d125 d59 p163 d13 p137 p114 d18 p140 p192 d134 p77 d27 d34 p40 p169 p190 d98 p51 d105 d104 d84 d26 p64 p183 d165 p61