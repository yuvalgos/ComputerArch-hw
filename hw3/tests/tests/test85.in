0 31 21 16
1 26 8 20
3 4 22 28
6 1 9 2
1 4 1 31
5 5 27 25
3 3 4 28
0 17 3 27
2 29 4 3
8 20 0 19
5 4 5 11
7 18 10 0
2 12 20 13
2 6 18 13
7 7 15 21
1 24 0 8
5 12 22 0
3 14 0 23
4 4 7 6
5 12 4 15
1 21 21 14
6 27 11 8
5 16 16 10
2 5 1 3
2 3 16 3
7 14 31 19
1 28 27 2
8 27 24 10
5 24 13 7
4 7 21 29
5 31 0 24
3 8 21 4
1 6 25 29
3 19 26 7
1 20 3 28
4 16 1 11
6 10 8 3
8 15 3 30
5 0 2 9
8 26 27 1
8 18 27 26
4 1 8 1
0 12 19 7
0 27 15 1
3 10 0 7
6 8 15 21
1 5 14 2
3 15 7 22
1 1 7 12
8 25 2 1
1 25 24 29
5 18 19 14
2 25 31 5
1 8 4 19
3 30 12 21
0 5 7 1
0 25 6 29
2 6 11 0
3 20 31 2
1 15 28 9
4 23 0 29
0 17 31 0
8 17 4 3
2 4 25 19
1 9 10 15
0 1 25 3
6 28 25 19
6 12 18 14
3 28 20 7
6 6 9 20
2 0 5 25
8 9 24 14
8 16 16 10
1 0 23 21
0 6 31 1
5 0 15 26
2 22 17 2
3 19 8 14
6 4 24 28
8 30 3 12
7 3 17 2
2 11 31 0
8 29 24 12
6 10 6 3
0 22 10 18
4 8 17 14
7 23 30 0
6 21 13 18
6 11 23 12
4 6 9 31
3 3 15 26
8 20 6 20
4 12 19 26
1 27 6 28
3 15 24 10
2 2 28 19
5 7 4 3
1 1 27 10
1 21 5 28
3 16 25 20
8 24 31 24
5 9 4 4
3 5 0 24
6 25 17 27
7 19 24 3
2 1 29 25
4 3 27 7
1 25 3 1
2 14 24 10
4 7 19 25
8 24 21 17
5 0 1 27
2 18 7 10
4 6 2 18
1 1 12 9
3 29 17 4
3 26 6 25
7 30 20 5
4 2 6 2
1 25 11 14
6 28 8 10
5 30 15 3
3 27 16 5
4 4 27 19
6 31 30 3
5 27 12 7
2 0 12 8
1 26 25 5
5 18 11 15
5 26 4 21
8 26 0 4
4 22 28 18
2 11 30 19
6 15 31 20
7 18 6 6
7 15 4 22
1 23 31 11
0 8 0 16
3 17 14 30
5 22 7 17
7 21 7 3
0 5 21 25
4 24 18 6
6 19 8 0
7 9 9 16
2 17 25 3
8 25 12 0
7 23 7 18
0 2 3 29
3 31 30 1
4 27 22 12
3 28 21 9
7 22 11 1
5 23 10 3
5 21 15 13
4 8 18 11
3 20 3 14
7 17 27 25
4 17 0 23
5 28 19 31
7 12 19 29
3 31 31 29
6 2 11 14
1 7 6 2
6 24 28 16
7 2 3 27
8 28 6 24
8 18 26 6
4 2 2 26
6 15 27 3
6 19 29 18
1 8 10 23
7 8 20 17
5 3 12 31
0 13 16 11
0 18 17 22
4 22 29 3
6 15 8 16
7 28 29 16
6 18 14 15
1 15 3 2
0 9 7 10
6 6 29 15
2 11 20 26
4 5 18 4
4 25 31 16
3 0 23 15
4 5 29 21
3 30 4 16
0 15 14 4
5 20 24 12
4 18 5 6
0 10 3 5
5 29 31 27
4 24 8 14
1 4 26 15
3 1 11 0
6 15 26 26
2 5 19 28
4 31 15 8
