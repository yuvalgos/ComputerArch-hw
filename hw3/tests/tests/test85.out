Reading the opcodes latency info from tests/test85.dat ... Got latency for 9 opcodes
Reading the program file tests/test85.in ... Found 200 instructions
getProgDepth()==217
getInstDeps(192)=={173,187}
getInstDeps(92)=={77,39}
getDepDepth(89)==55
getInstDeps(23)=={3,6}
getInstDeps(173)=={160,161}
getInstDeps(110)=={98,62}
getInstDeps(40)=={27,39}
getInstDeps(5)=={-1,-1}
getDepDepth(161)==121
getInstDeps(50)=={28,8}
getInstDeps(113)=={95,112}
getInstDeps(82)=={28,67}
getDepDepth(57)==13
getInstDeps(87)=={-1,51}
getDepDepth(110)==62
getInstDeps(124)=={121,106}
getInstDeps(9)=={-1,-1}
getDepDepth(80)==58
getInstDeps(97)=={93,83}
getInstDeps(103)=={62,93}
getDepDepth(124)==114
getInstDeps(78)=={28,68}
getDepDepth(44)==43
getInstDeps(37)=={24,-1}
getInstDeps(86)=={79,75}
getInstDeps(37)=={24,-1}
getInstDeps(63)=={56,33}
getDepDepth(16)==0
getInstDeps(111)=={105,93}
getInstDeps(121)=={94,106}
getInstDeps(172)=={156,158}
getInstDeps(77)=={53,25}
getInstDeps(156)=={106,108}
getDepDepth(12)==23
getDepDepth(31)==35
getDepDepth(36)==42
getInstDeps(90)=={59,39}
getInstDeps(130)=={126,123}
getDepDepth(6)==20
getInstDeps(168)=={165,130}
getDepDepth(145)==142
getInstDeps(34)=={24,26}
getDepDepth(94)==94
getDepDepth(32)==32
getInstDeps(180)=={173,168}
getInstDeps(80)=={62,-1}
getDepDepth(153)==100
getDepDepth(136)==131
getDepDepth(119)==139
getInstDeps(60)=={38,8}
getDepDepth(159)==121
getDepDepth(21)==0
getInstDeps(126)=={92,85}
getDepDepth(51)==50
getInstDeps(101)=={78,78}
getDepDepth(160)==115
getInstDeps(63)=={56,33}
getInstDeps(143)=={137,126}
getDepDepth(123)==103
getDepDepth(81)==62
getDepDepth(35)==17
getInstDeps(183)=={156,130}
getInstDeps(191)=={187,182}
getDepDepth(90)==49
getDepDepth(30)==32
getDepDepth(165)==146
getInstDeps(199)=={197,172}
getDepDepth(180)==176
getInstDeps(95)=={68,77}
getDepDepth(8)==27
getDepDepth(61)==45
getDepDepth(58)==45
getInstDeps(96)=={78,90}
getDepDepth(189)==114
getDepDepth(157)==146
getDepDepth(11)==0
getInstDeps(15)=={-1,-1}
getDepDepth(92)==60
getDepDepth(185)==128
getInstDeps(121)=={94,106}
getInstDeps(97)=={93,83}
getDepDepth(105)==103
getInstDeps(83)=={74,80}
getInstDeps(7)=={6,-1}
getDepDepth(41)==42
getInstDeps(40)=={27,39}
getDepDepth(40)==49
getInstDeps(31)=={20,18}
getInstDeps(55)=={29,48}
getDepDepth(155)==157
getDepDepth(76)==58
getInstDeps(72)=={35,44}
getDepDepth(66)==50
getDepDepth(44)==43
getInstDeps(112)=={109,83}
getInstDeps(31)=={20,18}
getDepDepth(145)==142
getInstDeps(135)=={123,131}
getDepDepth(28)==19
getDepDepth(150)==135
getInstDeps(175)=={158,152}
getDepDepth(41)==42
getInstDeps(6)=={4,-1}
getDepDepth(49)==55
getInstDeps(56)=={32,8}
getInstDeps(125)=={92,109}
getInstDeps(59)=={26,-1}
getDepDepth(163)==138
getInstDeps(13)=={11,-1}
getDepDepth(137)==96
getDepDepth(114)==89
getInstDeps(18)=={14,13}
getDepDepth(140)==122
getDepDepth(192)==206
getInstDeps(134)=={113,113}
getDepDepth(77)==53
getInstDeps(27)=={15,-1}
getInstDeps(34)=={24,26}
getDepDepth(40)==49
getDepDepth(169)==146
getDepDepth(190)==151
getInstDeps(98)=={55,68}
getDepDepth(51)==50
getInstDeps(105)=={82,103}
getInstDeps(104)=={100,90}
getInstDeps(84)=={83,51}
getInstDeps(26)=={21,-1}
getDepDepth(64)==50
getDepDepth(183)==137
getInstDeps(165)=={106,150}
getDepDepth(61)==45
